specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/bigDataPool.json

```diff
@@ -410,6 +410,10 @@
           "type": "boolean",
           "description": "Whether compute isolation is required or not."
         },
+        "sessionLevelPackagesEnabled": {
+          "type": "boolean",
+          "description": "Whether session support is required on library or not."
```

Comment:
"Whether session level package management is enabled or not."

Reply:
changed